model/project_parser_db.go

```diff
@@ -18,6 +18,7 @@ var (
 	ParserProjectEnabledKey           = bsonutil.MustHaveTag(ParserProject{}, "Enabled")
 	ParserProjectStepbackKey          = bsonutil.MustHaveTag(ParserProject{}, "Stepback")
 	ParserProjectPreErrorFailsTaskKey = bsonutil.MustHaveTag(ParserProject{}, "PreErrorFailsTask")
+	ParserProjectUnsetFunctionVarsKey = bsonutil.MustHaveTag(ParserProject{}, "UnsetFunctionVars")
```

Comment:
We don't need this key since we never use it.

Reply:
removed